CHANGELOG.md

```diff
@@ -25,7 +25,7 @@ Deprecated Features
 New Features
 ------------
 
-
+* Added new light distribution 'InclinedExponential'
```

Comment:
This is a little too terse (we are normally this terse in HISTORY.md, but the CHANGELOG.md usually has a bit more information).  Should include a sentence saying what this is meant to represent.  Also include issue number at the end so people know where to look for information.